### PR TITLE
Create diagram

### PR DESCRIPTION
**Pull request by samiullahbahadur**
## Project requirements

- [x] Created a file named `schema_based_on_diagram.sql` where we implemented the database from this [diagram](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/images/clinic_diagram.png).
- [x] Added `join tables` for many-to-many relationships.
- [x] Added the FK indexes.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	FOREIGN KEY (treatment_id) REFERENCES treatments(id),
+	
+	FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id)
+);
+
+--create indexes on foreign keys
+CREATE INDEX patient_id_asc ON medical_histories(patient_id ASC);
+CREATE INDEX medical_history_id_asc ON invoices(medical_history_id ASC);
+CREATE INDEX invoice_id_asc ON invoice_items(invoice_id ASC);
+CREATE INDEX treatment_id_asc ON invoice_items(treatment_id ASC);
```
Comment (mikethreels) [inline schema_based_on_diagram.sql]: - [ ] a table for invoice_items is not included, please include that as well

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+-- add table relationships
+-- create a "join table" called hospitalizations for the medical_histories and treatments tables
+
+CREATE TABLE hospitalizations(
+    ID int PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+	medical_history_id INT,
+	treatment_id INT,
+	FOREIGN KEY (treatment_id) REFERENCES treatments(id),
+	
+	FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id)
```
Comment (mikethreels) [inline schema_based_on_diagram.sql:45]: - [ ] there are no indexes created for these two foreign keys, please make sure to add them.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @samiullahbahadur, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**